File: Dart/File/get_file_size.dart
import "dart:core";
import "dart:io";

void main() {
  // Get the size of a given file.
  print(
    getFileSize("assets/valid/valid-json.json"),
  );
}

// Get the size of a given file.
int getFileSize(final String systemFile) {
  return File(systemFile).lengthSync();
}

File: TypeScript/File/get-file-size.ts
import * as fs from "fs"

function main(): void {
    // Get the size of a given path in bytes
    console.log(getFileSize("assets/valid/j3U5uJY779L49q98MX86iFsxs2kY9ew3/README.md"))
}

main()

// Get the size of a given file in bytes
function getFileSize(path: string): number {
    return fs.statSync(path).size
}